cibyl/models/ci/base/system.py

```diff
@@ -167,17 +165,14 @@ class JobsSystem(System):
     """Model a system with :class:`Job` as its top-level model.
     """
     API = deepcopy(System.API)
-    API.update({'jobs_scope': {'attr_type': str,
-                               'arguments': []},
-                'jobs': {
-                    'attr_type': Job,
-                    'attribute_value_class': AttributeDictValue,
-                    'arguments': [
-                        Argument(name='--jobs', arg_type=str,
-                                 nargs='*',
-                                 description="System jobs",
-                                 func='get_jobs')]
-    }})
+    API.update({
```

Comment:
API is missing jobs_scope now

Reply:
Yes, I'm not sure if it should be included there. It's more of a filter rather than an attribute/API. The problem is also if it's included there, its value gets reset.

Reply:
Oh right, since this is not something that will have a cli argument we don't need to have it in the API, sorry. We will have to make an slight change to the jenkins and elasticsearch sources to account for it though, since they relied on it being an argument

Reply:
Can you make the change? I'm not sure what should be changed tbh

Reply:
Sure, I'll push it